src/sql/pg/create.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS "L_ACCOUNTS" (
   "ACCOUNT_ID" SERIAL NOT NULL,
   "NAME" CHARACTER VARYING(255) NOT NULL,
-  "BALANCE" DECIMAL(32,16) NOT NULL CHECK
+  "BALANCE" DECIMAL(32,16) DEFAULT 0 NOT NULL CHECK
     ("BALANCE" >= "MINIMUM_ALLOWED_BALANCE"),
```

Comment:
Actually, I remembered that `MINIMUM_ALLOWED_BALANCE` must be nullable. The reason is, we use `NULL` in the db to correspond to `-infinity` (this is how we allow the balance to go negative without limit). See [here](https://github.com/interledger/five-bells-ledger/blob/77bdc25bc98f11a0bde6a7c472e5244a942b3fdc/src/models/db/accounts.js#L40).